templates/default/compute_ready.erb

```diff
@@ -10,4 +10,4 @@ instance_type_url="http://169.254.169.254/latest/meta-data/instance-type"
 instance_type=$(curl --retry 3 --retry-delay 0 --silent --fail ${instance_type_url})
 local_hostname_url="http://169.254.169.254/latest/meta-data/local-hostname"
 local_hostname=$(curl --retry 3 --retry-delay 0 --silent --fail ${local_hostname_url})
-<%= node['cfncluster']['cookbook_virtualenv_path'] %>/bin/aws --region ${cfn_region} sqs send-message --queue-url ${cfn_sqs_queue} --message-body '{"Type" : "Notification", "Message" : "{\"StatusCode\":\"Complete\",\"Description\":\"Succesfully launched '${instance_id}'\",\"Event\":\"parallelcluster:COMPUTE_READY\",\"EC2InstanceId\":\"'${instance_id}'\",\"Slots\":\"'${cfn_instance_slots}'\",\"LocalHostname\":\"'${local_hostname}'\",\"EC2InstanceType\":\"'${instance_type}'\"}"}'
+<%= node['cfncluster']['cookbook_virtualenv_path'] %>/bin/aws --region ${cfn_region} sqs send-message --queue-url ${cfn_sqs_queue} --endpoint https://sqs.${cfn_region}.amazonaws.com --message-body '{"Type" : "Notification", "Message" : "{\"StatusCode\":\"Complete\",\"Description\":\"Succesfully launched '${instance_id}'\",\"Event\":\"parallelcluster:COMPUTE_READY\",\"EC2InstanceId\":\"'${instance_id}'\",\"Slots\":\"'${cfn_instance_slots}'\",\"LocalHostname\":\"'${local_hostname}'\",\"EC2InstanceType\":\"'${instance_type}'\"}"}'
```

Comment:
Thanks for the contribution. The PR looks good to me but we need to handle the SQS endpoint in China regions as well. In that case the domain is `amazonaws.com.cn`. 

We could either extract the full endpoint from ${cfn_sqs_queue} that looks something like `https://sqs.us-east-2.amazonaws.com/123456789012/ab1-MyQueue-A2BCDEF3GHI4` or add some conditional logic such as `$([[ ${_region} =~ ^cn- ]] && echo ".cn")`